2021-05-13 11:39:57.127103: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory
2021-05-13 11:39:57.127400: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
Traceback (most recent call last):
  File "/home/antoine/dev/python/packages/timeseries/pyaf/tests/neuralnet/test_ozone_tensorflow.py", line 12, in <module>
    tf.python.control_flow_ops = tf
AttributeError: module 'tensorflow' has no attribute 'python'
